Sources/Kitura/RouterElement.swift

```diff
@@ -30,7 +30,11 @@ class RouterElement {
 
     /// The regular expression
     #if os(Linux)
-        private var regex: RegularExpression?
+        #if swift(>=3.1)
+            private var regex: NSRegularExpression?
+        #else
+            private var regex: RegularExpression?
+        #endif
```

Comment:
Possible to use `RegularExpressionType ` here to eliminate need for compiler directives? The typealias should be accessible anywhere inside the package.